### PR TITLE
Added code to prevent setup_test_environment() from being called twice.

### PR DESCRIPTION
**Pull request by cjerdonek**

### CODE REVIEW

File: django/test/utils.py [hunk 1]
```diff
@@ -101,6 +101,10 @@ def setup_test_environment():
         - Set the email backend to the locmem email backend.
         - Setting the active locale to match the LANGUAGE_CODE setting.
     """
+    if hasattr(Template, '_original_render'):
+        # Executing this function twice would overwrite the saved, pre-setup values.
+        raise Exception("setup_test_environment already called")
```
Comment (timgraham) [inline django/test/utils.py]: Maybe something like: `RuntimeError("setup_test_environment() was already called and can't be called more than once.")`

**Comment by timgraham** [comment]
Yes, this is a non-trivial change so it needs a ticket and also a mention in the release notes under "Backwards-incompatible changes".

**Comment by cjerdonek** [comment]
I created a ticket for this issue [here](https://code.djangoproject.com/ticket/27032)

**Comment by cjerdonek** [comment]
Amazing! The number of the ticket I just created is 27032, whose last four digits match the number of this pull request. 😉

**Comment by cjerdonek** [comment]
This is superseded by PR #7042.

**Pull request closed without merging**